Mathlib/MeasureTheory/Constructions/Prod/Integral.lean

```diff
@@ -234,9 +234,7 @@ theorem hasFiniteIntegral_prod_iff ⦃f : α × β → E⦄ (h1f : StronglyMeasu
     HasFiniteIntegral f (μ.prod ν) ↔
       (∀ᵐ x ∂μ, HasFiniteIntegral (fun y => f (x, y)) ν) ∧
         HasFiniteIntegral (fun x => ∫ y, ‖f (x, y)‖ ∂ν) μ := by
-  simp only [HasFiniteIntegral]
-  -- porting note (#10745): was `simp`
-  rw [lintegral_prod_of_measurable _ h1f.ennnorm]
+  simp only [HasFiniteIntegral, lintegral_prod_of_measurable _ h1f.ennnorm]
```

Comment:
The corresponding [mathlib3 line](https://github.com/leanprover-community/mathlib/blob/65a1391a0106c9204fe45bc73a039f056558cb83/src/measure_theory/constructions/prod/integral.lean#L215) was
`simp only [has_finite_integral, lintegral_prod_of_measurable _ h1f.ennnorm],`